eeps/eep-0044.md

```diff
@@ -3,7 +3,7 @@
             are implemented in OTP version 19.0
```

Comment:
in `eep-0045.md` the text says:

```
Status: Accepted/19.0  Implemented in OTP release 19
```

In this file `eep-0044.md` we say:

```
Status: Accepted/19.0-we  Proposal's -warning and -error directives
            are implemented in OTP version 19.0
```

We need consistency :) either 19.0 or 19.

Reply:
It should be updated to `release 19`, I missed this one.